Value Conversion Error on Import latitudesh_ssh_key.ssh_key

<!-- Thank you for opening an issue. Please note that we try to keep the issue
tracker reserved for bug reports and feature requests. For general usage
questions, please see:
https://github.com/latitudesh/terraform-provider-latitudesh/discussions

# Bug Report

<!-- Include as much of the following details with your bug report: -->
---

## Describe the bug
<!-- A clear and concise description of what the bug is. -->
Trying to import existing ssh keys and server definition into terraform state but encountering error that seems to be related to the tags.
```
│ Error: Value Conversion Error
│
│ An unexpected error was encountered while verifying an attribute value matched its expected type to prevent unexpected behavior or panics. This is always an error in the provider.
│ Please report the following to the provider developer:
│
│ Expected framework type from provider logic: types.ListType[basetypes.StringType] / underlying type: tftypes.List[tftypes.String]
│ Received framework type from provider logic: types.ListType[!!! MISSING TYPE !!!] / underlying type: tftypes.List[tftypes.DynamicPseudoType]
│ Path: tags
``` 

Import command
```
terraform import 'latitudesh_ssh_key.ssh_key["user1"]' proj_xxxx:ssh_xxxx
```

I also encounter a similar error for the `tags` on `latitudesh_server`. 


### Affected Resource(s)
<!-- Please list the resources, for example:
- latitudesh_ssh_key
- latitudesh_server

### Expected Behavior
The import should have succeeded with no errors.

### Actual Behavior
The import fails and I am not able to proceed.

### Steps to Reproduce
<!-- Please list the steps required to reproduce the issue, for example:
1. Manually create ssh_keys file via the latitude.sh dashboard
2. Write TF code for `latitudesh_ssh_key`. See below
3. Run `terraform import  'latitudesh_ssh_key.ssh_key["user1"]' proj_xxxx:ssh_xxxx
` command.

**Terraform Configuration Files**
<!--
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.-->
Here is the code for the ssh keys definition
```terraform
locals {
  ssh_keys = {
    user1 : "ssh-ed25519 xxxxxxxx foo"
  }
}

resource "latitudesh_ssh_key" "ssh_key" {
  for_each   = local.ssh_keys
  name       = each.key
  public_key = each.value
}
```

I have tried adding tags with multiple variations to the code but no luck
```
...
tags = []
...
tags = [""]
....
tags = ["test"]
```


**Terraform version**
<!-- Run `terraform -v` to show the version. If you are not running the latest
version of Terraform, please upgrade because your issue may have already been
fixed. -->
Terraform v1.11.4
on darwin_arm64
+ provider registry.terraform.io/latitudesh/latitudesh v2.1.0

**Debug Output**
<!-- Please provide a link to a GitHub Gist containing the complete debug output:
https://www.terraform.io/docs/internals/debugging.html. Please do NOT paste the
debug output in the issue; just paste a link to the Gist. -->
```
2025-07-07T16:19:33.386+0800 [ERROR] provider.terraform-provider-latitudesh_v2.1.0: Response contains error diagnostic: tf_proto_version=6.9 tf_provider_addr=registry.terraform.io/latitud
esh/latitudesh tf_resource_type=latitudesh_ssh_key diagnostic_severity=ERROR @module=sdk.proto diagnostic_attribute="AttributeName(\"tags\")"
  diagnostic_detail=
  | An unexpected error was encountered while verifying an attribute value matched its expected type to prevent unexpected behavior or panics. This is always an error in the provider. Ple
ase report the following to the provider developer:
  |
  | Expected framework type from provider logic: types.ListType[basetypes.StringType] / underlying type: tftypes.List[tftypes.String]
  | Received framework type from provider logic: types.ListType[!!! MISSING TYPE !!!] / underlying type: tftypes.List[tftypes.DynamicPseudoType]
  | Path: tags
   diagnostic_summary="Value Conversion Error" tf_req_id=dcf1fa2c-d360-2f75-ad66-02bf04db4435 @caller=github.com/hashicorp/terraform-plugin-go@v0.27.0/tfprotov6/internal/diag/diagnostics.
go:58 tf_rpc=ImportResourceState timestamp="2025-07-07T16:19:33.385+0800"
2025-07-07T16:19:33.387+0800 [ERROR] vertex "latitudesh_ssh_key.ssh_key[\"user1\"] (import id \"proj_xxxxxx:ssh_xxxxx\")" error: Value Conversion Error
2025-07-07T16:19:33.387+0800 [ERROR] vertex "latitudesh_ssh_key.ssh_key (expand)" error: Value Conversion Error
```

**Panic Output**
<!-- If Terraform produced a panic, please provide a link to a GitHub Gist
containing the output of the `crash.log`. -->

## Additional context
<!-- Add any other context about the problem here. -->

**References**
<!-- Include links to other GitHub issues (open or closed) or Pull Requests
that relate to this issue. -->

--- a/README.md
+++ b/README.md
@@ -75,7 +75,7 @@ terraform {
   required_providers {
     latitudesh = {
       source  = "latitude.sh/iac/latitudesh"
-      version = "2.1.0"
+      version = "2.1.1"
     }
   }
 }

--- a/docs/index.md
+++ b/docs/index.md
@@ -22,7 +22,7 @@ terraform {
   required_providers {
     latitudesh = {
       source  = "latitudesh/latitudesh"
-      version = "2.1.0"
+      version = "2.1.1"
     }
   }
 }

--- a/docs/resources/ssh_key.md
+++ b/docs/resources/ssh_key.md
@@ -24,7 +24,6 @@ resource "latitudesh_ssh_key" "ssh_key" {
 ### Required
 
 - `name` (String) The SSH key name
-- `project` (String, Deprecated) The id or slug of the project
 - `public_key` (String) The SSH public key
 
 ### Optional
@@ -38,8 +37,8 @@ resource "latitudesh_ssh_key" "ssh_key" {
 - `updated` (String) The timestamp for the last time the SSH key was updated
 
 ## Import
-SshKey can be imported using the sshKeyID along with the projectID that contains the sshKey, e.g.,
+The `latitudesh_ssh_key` resource can now be imported by specifying only the `sshKeyID`, for example:
 
 ```sh
-$ terraform import latitudesh_ssh_key.ssh_key projectID:sshKeyID
+$ terraform import latitudesh_ssh_key.ssh_key sshKeyID
 ```

--- a/examples/main.tf
+++ b/examples/main.tf
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     latitudesh = {
       source  = "latitudesh/latitudesh"
-      version = "2.1.0"
+      version = "2.1.1"
     }
   }
 }

--- a/latitudesh/provider.go
+++ b/latitudesh/provider.go
@@ -15,7 +15,7 @@ const (
 	userAgentForProvider = "Latitude-Terraform-Provider"
 )
 
-var currentVersion = "2.1.0"
+var currentVersion = "2.1.1"
 
 // Ensure latitudeshProvider satisfies various provider interfaces
 var _ provider.Provider = &latitudeshProvider{}

--- a/latitudesh/resource_server.go
+++ b/latitudesh/resource_server.go
@@ -574,6 +574,9 @@ func (r *ServerResource) readServer(ctx context.Context, data *ServerResourceMod
 		if attrs.Region != nil && attrs.Region.Site != nil && attrs.Region.Site.Slug != nil {
 			data.Region = types.StringValue(*attrs.Region.Site.Slug)
 		}
+
+		data.SSHKeys = types.ListNull(types.StringType)
+		data.Tags = types.ListNull(types.StringType)
 	}
 }
 

--- a/latitudesh/resource_ssh_key.go
+++ b/latitudesh/resource_ssh_key.go
@@ -256,6 +256,9 @@ func (r *SSHKeyResource) readSSHKey(ctx context.Context, data *SSHKeyResourceMod
 	}
 
 	sshKey := result.Object.Data
+
+	data.Tags = types.ListNull(types.StringType)
+
 	if sshKey.Attributes != nil {
 		if sshKey.Attributes.Name != nil {
 			data.Name = types.StringValue(*sshKey.Attributes.Name)

